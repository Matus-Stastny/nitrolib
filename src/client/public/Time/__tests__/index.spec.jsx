/* eslint-disable flowtype/no-types-missing-file-annotation */
import * as React from "react";
import { shallow } from "enzyme";

import WrappedTime from "../index";

const Time = WrappedTime.WrappedComponent;

describe("#Time", () => {
  it("should render correctly", () => {
    const time = new Date(Date.UTC(0, 0, 0, 10, 30));
    const wrapper = shallow(<Time time={time} locale="en" translate={id => id} />);

    expect(wrapper).toMatchSnapshot();
  });
});
