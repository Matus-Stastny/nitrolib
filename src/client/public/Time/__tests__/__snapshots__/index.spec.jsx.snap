// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Time should render correctly 1`] = `"11:30 AM"`;
